### DISCUSSION TITLE
Where to find the `model_checkpoint` for DetectorNode in webcam_demo.py

### DISCUSSION THREAD

**Discussion by dkobayas-cyber** [open-mmlab/mmpose#2094] [Q&A]
Thank you very much for developing the exciting tools! I have just started using the mmpose and have some questions about the configuration of `DetectorNode` in `webcam_demo.py`. 

As in [the document](https://github.com/open-mmlab/mmpose/blob/master/demo/docs/webcam_demo.md), the DetectorNode is configured like below: 
```
# 'DetectorNode':
# This node performs object detection from the frame image using an
# MMDetection model.
dict(
    type='DetectorNode',
    name='detector',
    model_config='demo/mmdetection_cfg/'
    'ssdlite_mobilenetv2_scratch_600e_coco.py',
    model_checkpoint='https://download.openmmlab.com'
    '/mmdetection/v2.0/ssd/'
    'ssdlite_mobilenetv2_scratch_600e_coco/ssdlite_mobilenetv2_'
    'scratch_600e_coco_20210629_110627-974d9307.pth',
    input_buffer='_input_',
    output_buffer='det_result')
```
I cannot fully understand these four lines: 
```
model_checkpoint='https://download.openmmlab.com'
    '/mmdetection/v2.0/ssd/'
    'ssdlite_mobilenetv2_scratch_600e_coco/ssdlite_mobilenetv2_'
    'scratch_600e_coco_20210629_110627-974d9307.pth'
```
My questions are: 

-  Which path does `model_checkpoint` point to? 

    If I understand correctly, this variable should specify a path or a URL to the model checkpoint (SSD-light model, in this case). However, I could not open `https://download.openmmlab.com` nor find a folder named `/mmdetection/v2.0/ssd/` in the github repo of `mmdetection`.  Where is the model file `scratch_600e_coco_20210629_110627-974d9307.pth` located? 

-  Where is the checkpoint of YOLOv3 model? 

    I found the model configuration file of YOLOv3 in `demo/mmdetection_cfg/` but cannot find its checkpoint.

**Comment by evansean** [comment]
the four lines make up the full URL to which you'll download the model checkpoint from.
